help/user-guide/index.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: FAIRDOM-SEEK User Guide
+title: User Guide
 redirect_from:
   - /help
 ---
 
-The FAIRDOM-SEEK user guide is written for users of any FAIRDOM-SEEK instance. The pages are organised by sections on the left. You can also search at the top-right of any page.
+The {{ site.seek_instance.name | default: "FAIRDOM-SEEK" }} user guide is written for users of any FAIRDOM-SEEK instance. The pages are organised by sections on the left. You can also search at the top-right of any page.
 
 
 ## Aliases for names of features and functions
```

Comment:
Would you mind checking the `/help/user-guide/aliases` page too, adding in any missing lines for DataHub? This page is deliberately not included in the side menu.